packages/ui/src/message-room-details-dialog/components/PrivateChannel.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import ChannelModal from "./ChannelModal";
+import styles from "./archive-channel.module.css";
+
+const PrivateChannel = ({ closeEdit }) => {
+  const org_id = localStorage.getItem("currentWorkspace");
+  const user = JSON.parse(sessionStorage.getItem("organisations"));
+  const room = window.location.href.split("/").at(6);
+  const BASE_URL = "https://chat.zuri.chat";
```

Comment:
Use the useParams hook provided by react to get this. What happens if the url changes. If it's not possible I think current room is stored in either local storage or session storage